ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -849,6 +859,111 @@ function getPoint(
     return point;
 }
 
+/**
+ * @private
+ */
+function scatterProcessData(
+    this: Series,
+    force?: boolean
+): (boolean|undefined) {
+    const series = this,
+        options = series.options,
+        xAxis = series.xAxis,
+        yAxis = series.yAxis;
+    (window as any).TheSeries = series;
+
+    // Process only on changes
+    if (
+        !series.isDirty &&
+        !xAxis.isDirty &&
+        !yAxis.isDirty &&
+        !force
+    ) {
+        return false;
+    }
+
+    // Required to get tick-based zoom ranges
```

Comment:
A little more elaboration? I _think_ the reason why we need to run the whole `setTickInterval` function is because it takes axis settings like `minPadding`, `maxPadding`, `startOnTick` and `endOnTick` into account. It is hard to account for these without actually running the whole function.